app/views/purchase_or_donate_mailer/purchase_donate_email.html.slim

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
http://www.rubydoc.info/gems/slim/frames

slim gets confused if you just put actual html in the file